jujugui/static/gui/src/app/components/added-services-list-item/test-added-services-list-item.js

```diff
@@ -25,12 +25,44 @@ chai.config.includeStack = true;
 chai.config.truncateThreshold = 0;
 
 describe('AddedServicesListItem', function() {
+  var mockService;
 
   beforeAll(function(done) {
     // By loading this file it adds the component to the juju components.
     YUI().use('added-services-list-item', function() { done(); });
   });
 
+  beforeEach(function() {
+    mockService = {
```

Comment:
Lets move this to `component-test-utils.js` so that we can enhance/reuse elsewhere.